build.gradle

```diff
@@ -110,7 +110,8 @@ allprojects {
     dependencies {
         implementation "com.google.guava:guava:32.1.2-jre"
         implementation 'com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.15.0'
-
+        // jackson-core is a transitive dep coming from jackson-dataformat-yaml, earlier versions have vulnerability sonatype-2022-6438
+        implementation('com.fasterxml.jackson.core:jackson-core:2.15.0')
```

Comment:
We have to add it as a direct dependency here in order to override a version constraint coming from a global dependency management configuration

Reply:
I suppose just adding a direct dependency is sufficient and we don't need to exclude the transitive too from `io.spring.dependency-management` in addition to it?

Reply:
Correct, I see all other versions of Jackson-core be corrected to --> 2.15.0 in the dependency tree. Other attempts always resulted in 2.13.5 winning (which I believe is coming from spring dependency management). I have also confirmed that in the Detect JAR built from the branch, only version 2.15.0 is present.

Reply:
I see, thanks for explaining